src/trackers/build-trackers.js

```diff
@@ -4,14 +4,23 @@ const chalk = require('chalk')
 const sharedData = require('./helpers/sharedData.js')
 const Progress = require('progress')
 
-const newData = JSON.parse(fs.readFileSync(`${sharedData.config.trackerDataLoc}/commonRequests.json`, 'utf8'))
+const newDataStats = JSON.parse(fs.readFileSync(`${sharedData.config.trackerDataLoc}/crawlStats.json`, 'utf8'))
+let requestsArray = []
+
+let chunk = 0
+while (fs.existsSync(`${sharedData.config.trackerDataLoc}/commonRequests-${chunk}.json`)) {
+    const requests = JSON.parse(fs.readFileSync(`${sharedData.config.trackerDataLoc}/commonRequests-${chunk}.json`, 'utf8'))
+    requestsArray = requestsArray.concat(requests)
+    chunk++
+}
```

Comment:
commonRequests was getting too big, so I chunked it into multiple files 50k requests each

Reply:
Can you clean up any old files before building a new data set? You can run into problems if you were to process a smaller crawl (fewer split files) after processing a larger crawl (more split files).

Reply:
Ah, great catch. Done!